CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- Add proxy_mode
```

Comment:
These should be under an `### Added` header. Also the description says fixed strip_keys keys but this is not represented here. Please add it under a `### Fixed Header`